app/src/js/config/config.js

```diff
@@ -1,10 +1,23 @@
+/**
+ * computeBool
+ * @description Takes string env variables and converts them to the intended bool value
+ * @param {string} value value to convert to boolean
+ * @param {boolean || string} defaultValue default value for variable if none provided by user
+ * @returns boolean value or default
+ */
+function computeBool (value, defaultValue) {
+  if (!value) return defaultValue;
+  if (typeof value !== 'string') return value;
+  return value && value.toLowerCase() === 'true';
```

Comment:
Using the defaultValue here because otherwise if the value passed in is `false`, then it will automatically fall back to the defaultValue. We want to use `false` when it is passed in by the user